src/it/java/org/jboss/tools/intellij/quarkus/BasicTests.java

```diff
@@ -40,22 +40,22 @@ public static void connect() throws InterruptedException {
 
     @AfterEach
     public void finishTheTestRun() {
-        GlobalUtils.checkForExceptions(robot);
```

Comment:
What is the rationale for moving out these checks ?

Reply:
The **createNewQuarkusProjectWithInvalidCustomEndpointURL** test did not created new project, the **checkForExceptions** method checks for exceptions only from the open project. I see 2 option - leave it as it is or rework the **checkForExceptions** method to check the exception always from the 'Welcome to IntelliJ IDEA' dialog (which may be better, it will always check for exceptions and it will stay in the AfterEach).

Reply:
I think the less we duplicate code the better is it so +1 for reworking **checkForExceptions**

Reply:
Reworked.